src/content/h1/tools/hek/sapien/readme.md

```diff
@@ -4,11 +4,18 @@ toolName: Sapien
 stub: true
 img: netgame_equipment_04_large.gif
 imgCaption: Sapien being used to place item spawns in `tutorial.map`
+thanks:
+  - to: Real Fake Doors
+    for: Sharing NT Core knowledge
 ---
 Sapien is a visual editing tool used to place objects in [scenarios][scenario].
 
 # Compatibility
 On Linux, Sapien can be run successfully using [Wine][] but is not yet compatible with [DXVK][]. Use built-in or standard native DirectX libraries instead.
 
+# Limits
+As an older 32-bit Windows application, Sapien is limited to 2GB of virtual memory even on modern 64-bit Windows systems for compatibility. While this memory limit is usually not an issue, an abundance of large textures and other large assets in a map may cause Sapien to crash. To work around this, `sapien.exe` can be patched to tell the OS it supports 4GB of virtual memory using a utility like [NTCore][ntcore].
```

Comment:
We should put in a todo to have some kind of guide on how to patch Sapien for this